### PR TITLE
[Refactor]-Filter main page

### PR DESCRIPTION
**Pull request by subhradwip01**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryFilters.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    ("grid")
+  );
+
+  function resetSelectedRowData() {
+    return () => {
+      setSelectedRowData(null);
+    };
+  }
+
+  function fetchfilters(page, pageSize, search, sortOrder) {
```
Comment (YashKamboj) [inline ui/components/MesheryFilters.js:61]: don't make fetchfilters for now  @subhradwip01

**Review by YashKamboj — CHANGES_REQUESTED** [review]

**Comment by Abhishek-kumar09** [comment]
@subhradwip01 Please post a screenshot...

**Comment by subhradwip01** [comment]
> @subhradwip01 Please post a screenshot...

Ok..I will post it by tonight

**Comment by subhradwip01** [comment]
> @subhradwip01 Please post a screenshot...

Ok..I will post it by tonight

**Comment by subhradwip01** [comment]
Screenshot from 2022-09-14 01-40-05
Currently don't have any data..as I am not fetching data from the server..

**Review by Abhishek-kumar09 — APPROVED** [review]
The checkpoint seems Fine

**Pull request merged**